src/component/project/Project/ProjectForm/ProjectForm.tsx

```diff
@@ -0,0 +1,102 @@
+    setProjectId: React.Dispatch<React.SetStateAction<string>>;
+    setProjectName: React.Dispatch<React.SetStateAction<string>>;
+    setProjectDesc: React.Dispatch<React.SetStateAction<string>>;
+    handleSubmit: (e: any) => void;
+    handleCancel: () => void;
+    errors: { [key: string]: string };
+    submitButtonText: string;
+    clearErrors: () => void;
+    validateIdUniqueness: () => void;
+}
+
+const ProjectForm = ({
+    handleSubmit,
+    handleCancel,
+    projectId,
+    projectName,
+    projectDesc,
+    setProjectId,
+    setProjectName,
+    setProjectDesc,
+    errors,
+    submitButtonText,
+    validateIdUniqueness,
+    clearErrors,
+}: IProjectForm) => {
+    const styles = useStyles();
+
+    return (
+        <form onSubmit={handleSubmit} className={styles.form}>
+            <h3 className={styles.formHeader}>Project Information</h3>
+
+            <div className={styles.container}>
+                <p className={styles.inputDescription}>
+                    What is your project Id?
+                </p>
+                <Input
+                    className={styles.input}
+                    label="Project Id"
+                    value={projectId}
+                    onChange={e => setProjectId(e.target.value)}
+                    error={Boolean(errors.id)}
+                    errorText={errors.id}
+                    onFocus={() => clearErrors()}
+                    onBlur={validateIdUniqueness}
+                    disabled={submitButtonText === 'Edit'}
+                />
+
+                <p className={styles.inputDescription}>
+                    What is your project name?
+                </p>
+                <Input
+                    className={styles.input}
+                    label="Project name"
+                    value={projectName}
+                    onChange={e => setProjectName(e.target.value)}
+                    error={Boolean(errors.name)}
+                    errorText={errors.name}
+                    onFocus={() => clearErrors()}
+                />
+
+                <p className={styles.inputDescription}>
+                    What is your project description?
+                </p>
+                <TextField
+                    className={styles.input}
+                    label="Project description"
+                    variant="outlined"
+                    multiline
+                    maxRows={4}
+                    value={projectDesc}
+                    onChange={e => setProjectDesc(e.target.value)}
+                />
+            </div>
+
+            <div className={styles.buttonContainer}>
+                <Button onClick={handleCancel} className={styles.cancelButton}>
+                    Cancel
+                </Button>
+                <PermissionButton
+                    onClick={handleSubmit}
+                    permission={ADMIN}
+                    type="submit"
+                >
+                    {submitButtonText} project
```

Comment:
This should no longer be needed. We have separate components now for Create and Edit.